frontend/src/components/elements/Video/Video.test.tsx

```diff
@@ -98,26 +98,16 @@ describe("Video Element", () => {
 
   describe("updateTime", () => {
     const props = getProps()
-    const wrapper = shallow(<Video {...props} />)
-    const instance = wrapper.instance()
+    const wrapper = mount(<Video {...props} />)
+    const videoElement: HTMLVideoElement = wrapper.find("video").getDOMNode()
 
-    // @ts-ignore
-    jest.spyOn(instance, "updateTime")
-
-    it("should call it on didMount", () => {
-      // @ts-ignore
-      instance.componentDidMount()
-
-      // @ts-ignore
-      expect(instance.updateTime).toBeCalledTimes(1)
+    it("current time should be set to startTime on mount", () => {
```

Comment:
@tanmaylaud Can you please change this to be in the following format:
`it("should set current time to startTime on mount")`?

Reply:
Done. thanks!